dnf5/commands/history/arguments.hpp

```diff
@@ -41,6 +44,30 @@ class ReverseOption : public libdnf5::cli::session::BoolOption {
         : BoolOption(command, "reverse", '\0', _("Reverse the order of transactions."), false) {}
 };
 
+class IgnoreInstalledOption : public libdnf5::cli::session::BoolOption {
+public:
+    explicit IgnoreInstalledOption(libdnf5::cli::session::Command & command)
+        : BoolOption(
+              command,
+              "ignore-installed",
+              '\0',
+              _("Don't consider mismatches between installed and stored transaction packages as errors."),
```

Comment:
I am thinking whether the description is clear enough. The first part is OK, but `stored transaction packages` is somehow unclear. It does not explain what in stored transaction is related and how the mismatch will be handled. For upgrade it will replace currently installed packages, but for remove step it will skip remove completely. I think that detailed description can be in man pages.

Reply:
The hint is perfect, I love it, and can be used as an inspiration. 

```
Cannot perform Remove action because 'zsh-html-5.9-8.fc39.noarch' is installed in a different version: 'zsh-html-5.9-10.fc39.noarch'.
You can try to add to command line:
  --ignore-installed to allow mismatches between installed and stored transaction packages. This can result in an empty transaction because among other things the option can ignore failing Remove actions
```

Reply:
I have updated the description.